src/components/TradeBox/TradeBox.js

```diff
@@ -219,7 +219,7 @@ const TradeBox = ({
 					<TradeInput
 						synth={base.name}
 						amount={baseAmount}
-						onAmountChange={value => {
+						onChange={(_, value) => {
```

Comment:
what's this?

Reply:
`_` is just a convention for "I'm not going to use the first variable" (which is the event)